README.md

```diff
@@ -48,7 +48,7 @@ the following:
     EHL/foiBKY2Ucy3oYmxdkFiXzWnOu7by
 
 The 'input' is your database password. The 'password' is a password specified by you that can be used to decrypt the data.
-The 'output' is your encrypted password. This encrypted password can then be used in the Circus Train yaml configuration
+The 'output' is your encrypted password. This encrypted password can then be used in the  yaml configuration
```

Comment:
Hmmm, we don't have any YAML here so I guess remove the reference to it and the snippet below. If/when we have a CLI tool as part of this project we can add documentation for it that explains how to configure it.